SPECS/bazel/bazel.spec

```diff
@@ -1,37 +1,33 @@
+%define _enable_debug_package 0
+%global debug_package %{nil}
+%define __os_install_post %{_libdir}/rpm/brp-compress %{nil}
```

Comment:
Do I understand your intention correctly, that you don't want to generate the `bazel-debuginfo` subpackage but also prevent the symbols from being stripped? If so, and you know this works, could you let Andrew know? He's currently working on doing exactly that, but perhaps with a single macro.

Reply:
Yes, that's correct.

Reply:
Related question: why we don't want to build the `bazel-debuginfo` subpackage in this case? Usually, we try to preserve these, if possible. Could you add a comment with an explanation at the top of these macros?

Reply:
bazel built with stripping will cause error when using the bazel binary. Refer to this link: https://github.com/bazelbuild/bazel/issues/600. And a common solution to resolve this issue is to disable stripping and creation of the debug package.

Reply:
Interesting, thanks for the link!